app/containers/TwoFactor/styles.ts

```diff
@@ -37,9 +38,23 @@ export default StyleSheet.create({
 	},
 	buttonContainer: {
 		flexDirection: 'row',
-		justifyContent: 'space-between'
+		justifyContent: 'space-between',
+		marginTop: 36
 	},
 	tablet: {
 		height: undefined
+	},
+	overlay: {
+		flex: 1,
+		alignItems: 'center',
+		justifyContent: 'center'
+	},
+	containerInput: {
+		marginBottom: 0
+	},
+	resendEmailText: {
```

Comment:
If there's a `small` prop on Button, why do you have to send an additional style? 🙈 
Can you fix there instead?

Reply:
got it, adjusted.